script/Ciel/Day 13/13_01_02_CIEL13_7.txt

```diff
@@ -341,27 +390,33 @@ That's right, I've said it! Take that!
 [sha:7915d30b63c8b25ce104883f4767f2621d77e32e]{
 -- Page 225, Offset 35906. Glued. Mods: @x.
 -- 拷問って、苦しめるだけが答えじゃないのね。命を奪うコト。未来を奪うコトそのものが責め苦になるヤツも、いるワケ」
-Inflicting pain isn't the only form of torture, you know? Taking away one's life. Taking away one's future. Those are also forms of torture."
+ Inflicting pain isn't the only form of torture, you know?
+ Taking away one's life.
+ Taking away one's future.
+ Those are also forms of torture."
```

Comment:
one's > someone's

I think that works better here. Otherwise it feels to reflexive.